release.Rmd

```diff
@@ -26,21 +63,304 @@ To get your package ready to release, follow these steps:
 6.  Prepare for the next version by updating version numbers.
 7.  Publicise the new version.
 
-## The submission process {#sec-release-process}
+## Decide the release type
 
-To manually submit your package to CRAN, you create a package bundle (with `devtools::build()`) then upload it to <https://cran.r-project.org/submit.html>, along with some comments which describe the process you followed.
-This section shows you how to make submission as easy as possible by providing a standard structure for those comments.
-Later, in @sec-release-submission, you'll see how to actually submit the package with `devtools::release()`.
+When you call `use_release_issue()`, you'll be asked which type of release you intend to make (@sec-lifecycle-release-type).
 
-When submitting to CRAN, remember that CRAN is staffed by volunteers, all of whom have other full-time jobs.
-A typical week has over 100 submissions and only three volunteers to process them all.
-The less work you make for them the more likely you are to have a pleasant submission experience.
+```{r}
+#| eval: false
+> use_release_issue()
+✔ Setting active project to '/Users/jenny/rrr/usethis'
+Current version is 2.1.6.9000.
+What should the release version be? (0 to exit) 
+
+1: major --> 3.0.0
+2: minor --> 2.2.0
+3: patch --> 2.1.7
+4:   dev --> 2.1.6.9001
+
+Selection: 
+```
+
+This really boils down to which component of the version number is going to be incremented.
+In our workflow, this planned version number is recorded in the GitHub issue that holds the release checklist, but we don't actually increment the version in `DESCRIPTION` until later in the process.
+
+## Initial CRAN release: Special considerations
+
+Every new package receives a higher level of scrutiny from CRAN.
+In addition to the usual automated checks, new packages are also reviewed by a human, which inevitably introduces a certain amount of subjectivity and randomness.
+There are many packages on CRAN that would not be accepted in their current form, if submitted today as a completely new package.
+This isn't meant to discourage you.
+But you should be aware that, just because you see some practice in an established package (or even in base R), that doesn't mean you can do the same in your new package.
+There's a reason the community maintains lists of common "gotchas" for new packages; we have all learned the hard way.
```

Comment:
I feel like this could do with a bit more positivity: e.g. in the tidyverse, we follow this checklist and recently the majority of our packages have been accepted on first try; this has not always been the case.

Reply:
done